pkgs/by-name/nr/nrc-wrapper-go/package.nix

```
@@ -0,0 +1,31 @@
+{
+  fetchFromGitHub,
+  lib,
+  buildGoModule,
+}:
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```
this way you can see exactly where attrs come from + can reuse implicit attrs (such as `finalAttrs.finalPackage`, used in some passthrus)